SPEC/ovirt-openvswitch.spec

```diff
@@ -0,0 +1,153 @@
+%define ovs_version 2.11
+%define ovs_version_to_replace 2.11.1
+
+%global py_openvswitch python3-openvswitch
+
+Name:           ovirt-openvswitch
+Version:        %{ovs_version}
+Release:        0.2020060501%{?dist}
+Summary:        Wrapper RPM for upgrading OVS to newer versions
+
+Group:          System Environment/Daemons
+License:        Public Domain
+URL:            http://www.openvswitch.org
+BuildArch:      noarch
+
+Requires:       openvswitch%{ovs_version}
+Provides:       openvswitch = %{ovs_version}
+Obsoletes:      openvswitch <= %{ovs_version_to_replace}
+Obsoletes:      openvswitch-dpdk <= %{ovs_version_to_replace}
+
+%description
+Wrapper rpm for the openvswitch package
+
+%package -n     ovirt-python-openvswitch
+Summary:        Wrapper for python-openvswitch rpm
+License:        Public Domain
+Requires:       %{py_openvswitch}%{ovs_version}
+Provides:       %{py_openvswitch} = %{ovs_version}
+Obsoletes:      %{py_openvswitch} <= %{ovs_version_to_replace}
+
+%description -n ovirt-python-openvswitch
+Wrapper rpm for the base python-openvswitch package
+
+%package        devel
+Summary:        Wrapper for openvswitch-devel rpm
+License:        Public Domain
+Requires:       openvswitch%{ovs_version}-devel
+Provides:       openvswitch-devel = %{ovs_version}
+Obsoletes:      openvswitch-devel <= %{ovs_version_to_replace}
+
+%description devel
+Wrapper rpm for the base openvswitch-devel package
+
+%package        ovn
+Summary:        Wrapper for ovn rpm
+License:        Public Domain
+Requires:       ovn%{ovs_version}
+Provides:       openvswitch-ovn = %{ovs_version}
+Provides:       ovn = %{ovs_version}
+Obsoletes:      ovn <= %{ovs_version_to_replace}
+
+%description ovn
+Wrapper rpm for the base openvswitch-ovn-central package
+
+%package        ovn-central
+Summary:        Wrapper for openvswitch-ovn-central rpm
+License:        Public Domain
+Requires:       ovn%{ovs_version}-central
+Requires:       ovirt-openvswitch-ovn
```

Comment:
who's providing this ?

Reply:
The resulting name of the ovn package 15 lines above is ovirt-openvswitch-ovn .